spec/lib/email/receiver_spec.rb

```diff
@@ -373,6 +373,12 @@ def create_post_reply_key(value)
       expect(IncomingEmail.last.created_via).to eq(IncomingEmail.created_via_types[:imap])
     end
 
+    it "stores the message_id of the incoming email against the post as outbound_message_id" do
+      process(:text_reply, source: :handle_mail)
```

Comment:
```suggestion
      expect { process(:text_reply, source: :handle_mail) }.to change(Post, :count)
```

Helps to make it clear that this creates a post.